nncf/common/quantization/quantizer_propagation/graph.py

```diff
@@ -973,6 +978,8 @@ def get_visualized_graph(self):
             is_integer_path = edge[QuantizerPropagationStateGraph.IS_INTEGER_PATH_EDGE_ATTR]
             if is_integer_path:
                 attrs = {"color": "violet", "style": "bold"}
+            u = u.replace(__RESERVED_DOT_CHARACTER, __CHARACTER_REPLACE_TO)
+            v = v.replace(__RESERVED_DOT_CHARACTER, __CHARACTER_REPLACE_TO)
```

Comment:
Really, all of this needs to be done at one place - maybe a function that takes an NX graph and then examines all of its attribute nodes or labels and does the replacement.

Reply:
Moved to separate PR https://github.com/openvinotoolkit/nncf/pull/2231